larray/io/array.py

```diff
@@ -324,6 +324,12 @@ def read_excel(filepath, sheetname=0, nb_index=None, index_col=None, na=np.nan,
         if kwargs:
             raise TypeError("'{}' is an invalid keyword argument for this function when using the xlwings backend"
                             .format(list(kwargs.keys())[0]))
+        if not np.isnan(na):
+            raise NotImplementedError("na argument is not currently supported with xlwings engine")
```

Comment:
"with the default engine (xlwings)" (or something similar)?

Reply:
"with the (default) xlwings engine"?

Reply:
Sure.